test/integration/emitter_test.cpp

```diff
@@ -1212,6 +1212,42 @@ TEST_F(EmitterTest, NaN) {
 	  "bar: .nan");
 }
 
+TEST_F(EmitterTest, ComplexFlowSeqEmbeddingAMapWithNewLine) { 
```

Comment:
Could you add a test similar to this example, but with an alias (or a tag)? That would help determine whether this is a correct change.

Reply:
i have done with alias;
for the tag test, i notice there is no too much documentation neither and it makes me quite difficult translate this minimal test to an equivalent using a tag (local?), basically i need some help/ more information around YAML tags to do it.

Reply:
Yeah, I think just using `LocalTag("foo")` should work fine; it should produce `!foo` as a prefix to whichever node you're applying it to.